query/sources/impl/src/main/java/org/codice/ddf/admin/sources/csw/CswSourceUtils.java

```diff
@@ -96,10 +97,14 @@ public CswSourceUtils(RequestUtils requestUtils, SourceUtilCommons sourceUtilCom
      */
     public ReportWithResult<ResponseField> discoverCswUrl(HostField hostField,
             CredentialsField creds) {
-        return requestUtils.discoverUrlFromHost(hostField,
+        ReportWithResultImpl<ResponseField> responseResult = requestUtils.discoverUrlFromHost(hostField,
                 URL_FORMATS,
                 creds,
                 GET_CAPABILITIES_PARAMS);
+        if(responseResult.containsErrorMsgs()) {
```

Comment:
Even though I changed the `RequestUtil` to return exactly what will return if this is true, I didn't want this breaking if there are any changes to `RequestUtil` later on, since this is what we always want for this action and `RequestUtils` are in another module.